src/integrationTest/java/org/opensearch/test/framework/TestSecurityConfig.java

```diff
@@ -449,9 +449,9 @@ public int hashCode() {
 
     public static final class User implements UserCredentialsHolder, ToXContentObject {
 
-        public final static TestSecurityConfig.User USER_ADMIN = new TestSecurityConfig.User("admin").roles(
+        public final static TestSecurityConfig.User USER_ADMIN = new User("admin").roles(
             new Role("allaccess").indexPermissions("*").on("*").clusterPermissions("*")
-        );
+        ).backendRoles("admin");
```

Comment:
This is a global constant that can be referenced by all tests that provides a reusable admin user. As the user has a predefined role which gives full privileges on all indices and cluster permissions, the additional backend role should be not necessary. Also, adding a backend role here makes the constant actually less reusable because backend roles are always subject to role mapping. Even though it is named "admin", you'll never know wether the actually exists a role to which it is mapped or not.

Reply:
This change is required to test case where resources are default shared with user's backend-roles. AD is one example that does this. But I have modified the test to create a local user, instead of using USER_ADMIN.